airflow/runtimes/container_runtime.go

```diff
@@ -1,17 +1,59 @@
-package airflow
+package runtimes
 
 import (
 	"os"
 	"path/filepath"
 	"strings"
 	"sync"
+	"time"
+
+	"github.com/briandowns/spinner"
 
 	"github.com/astronomer/astro-cli/config"
 	"github.com/astronomer/astro-cli/pkg/fileutil"
 	"github.com/astronomer/astro-cli/pkg/util"
 	"github.com/pkg/errors"
 )
 
+var spinnerCharSet = spinner.CharSets[14]
+
+const (
+	docker                         = "docker"
```

Comment:
Do we need these verbatim constants? I'd think if the value changed we'd probably just rename the constant anyway.

Reply:
What are you thinking? Change them back to `dockerCmd`, etc?

Reply:
Could just use the literal string.